utils/send_email.py

```diff
@@ -69,6 +69,23 @@ def taxpayer_notification(taxpayer, change_type):
     send_email_notification.apply_async([subject, message, recipient_list])
 
 
+def buyer_notification(taxpayer, change_type):
+    subject = _(email_notifications[change_type]['subject'])
+    upper_text = _(email_notifications[change_type]['body']['upper_text'])
+    lower_text = _(email_notifications[change_type]['body']['lower_text'])
+    btn_text = _(email_notifications[change_type]['body']['btn_text'])
+    btn_url = email_notifications[change_type]['body']['btn_url']
+    message = build_mail_html(
+        taxpayer.business_name,
+        upper_text,
+        lower_text,
+        btn_text,
+        btn_url,
+    )
```

Comment:
This piece of code is the same that 56-67
Why don't make a function merging the code repeated?

```
def get_message_and_subject(change_type: str, taxpayer: TaxPayer) -> Tuple[str, str]:
    subject = _(email_notifications[change_type]['subject'])
    upper_text = _(email_notifications[change_type]['body']['upper_text'])
    lower_text = _(email_notifications[change_type]['body']['lower_text'])
    btn_text = _(email_notifications[change_type]['body']['btn_text'])
    btn_url = email_notifications[change_type]['body']['btn_url']
    message = build_mail_html(
        taxpayer.business_name,
        upper_text,
        lower_text,
        btn_text,
        btn_url,
    )
    return message, subject


def taxpayer_notification(taxpayer, change_type):
    message, subject = get_message_and_subject(change_type, taxpayer)
    recipient_list = get_user_emails_by_tax_payer_id(taxpayer.id)
    send_email_notification.apply_async([subject, message, recipient_list])


def buyer_notification(taxpayer: TaxPayer, change_type: str):
    message, subject = get_message_and_subject(change_type, taxpayer)
    recipient_list = get_buyer_emails_by_tax_payer_id(taxpayer.id)
    send_email_notification.apply_async([subject, message, recipient_list])
```